[2024-05-07 06:13:17 +0000] [97453] [INFO] Starting gunicorn 22.0.0
[2024-05-07 06:13:17 +0000] [97453] [INFO] Listening at: http://127.0.0.1:8000 (97453)
[2024-05-07 06:13:17 +0000] [97453] [INFO] Using worker: sync
[2024-05-07 06:13:17 +0000] [97454] [INFO] Booting worker with pid: 97454
[2024-05-07 06:13:17 +0000] [97454] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
    worker.init_process()
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/util.py", line 371, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'hcenter.wsgi'
[2024-05-07 06:13:17 +0000] [97454] [INFO] Worker exiting (pid: 97454)
[2024-05-07 06:13:17 +0000] [97453] [ERROR] Worker (pid:97454) exited with code 3
[2024-05-07 06:13:17 +0000] [97453] [ERROR] Shutting down: Master
[2024-05-07 06:13:17 +0000] [97453] [ERROR] Reason: Worker failed to boot.
[2024-05-07 06:13:18 +0000] [97455] [INFO] Starting gunicorn 22.0.0
[2024-05-07 06:13:18 +0000] [97455] [INFO] Listening at: http://127.0.0.1:8000 (97455)
[2024-05-07 06:13:18 +0000] [97455] [INFO] Using worker: sync
[2024-05-07 06:13:18 +0000] [97456] [INFO] Booting worker with pid: 97456
[2024-05-07 06:13:18 +0000] [97456] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/arbiter.py", line 609, in spawn_worker
    worker.init_process()
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/var/www/gel/hcenter_back/hcenter/env/lib/python3.10/site-packages/gunicorn/util.py", line 371, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'hcenter.wsgi'
[2024-05-07 06:13:18 +0000] [97456] [INFO] Worker exiting (pid: 97456)
[2024-05-07 06:13:18 +0000] [97455] [ERROR] Worker (pid:97456) exited with code 3
[2024-05-07 06:13:18 +0000] [97455] [ERROR] Shutting down: Master
[2024-05-07 06:13:18 +0000] [97455] [ERROR] Reason: Worker failed to boot.
